app/models/course/assessment/question/text_response.rb

```diff
@@ -80,6 +80,15 @@ def build_at_least_one_group_one_point
     groups.first.points.build if groups.first.points.empty?
   end
 
+  # returns the type of question as text response or file upload
+  def type
```

Comment:
Actually this method has already been implemented here, see `#question_type`.

Reply:
I'll use that. Thank you.